number-filtering.js

```diff
@@ -0,0 +1,66 @@
+let myNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+let numbers = myNumbers;
+
+function filterEven(){
+    for(let num of numbers){
+      if(num % 2 == 0){
+        console.log(num)
+      }
+    }
+}
+
+filterEven();

```

Comment:
this will console out each individual number, the even numbers should be in an array